lib/email_validator.rb

```diff
@@ -14,6 +14,7 @@ def validate(record)
 
   private
 
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Why're we disabling the check here? Wouldn't it make more sense to follow the pattern you use in `WebhooksController`?